### DISCUSSION TITLE
Open gallery tabs without switching to them?

### DISCUSSION THREAD

**Discussion by darkred3** [Difegue/LANraragi#591] [Ideas]
The _Read_ button of the right click menu should be changed to

- [x]  Be placed at the very top instead of second to last. Every browser I know of (Firefox, Chrome) does exactly this, because opening a link in another tab is the most used option of the right click menu.
- [ ]  Open the tab as a "background tab" instead of "foreground tab"

![rightclickmenu](https://user-images.githubusercontent.com/33666486/113911537-63ebe200-97da-11eb-82fe-dd087f0272ae.jpg)

**Comment by Difegue** [comment]
Both of those are pretty easy, it's just a matter of:
* moving the admin options block after Read/download in index.html.tt2's context menu  
* Use the openInNewTab() JS function for the read button instead of the location.href that's probably in there right now.

**Comment by Difegue** [comment]
Fixed. 👏

**Comment by darkred3** [comment]
@Difegue Been almost a year and I totally forgot to ever reply lol. The archives still open in foreground, can this be fixed?

**Comment by Asinin3** [comment]
Just hold down ctrl when you click open gallery and it will open a tab instead?

**Comment by Difegue** [comment]
Oh right, I thought "background tab" meant just opening a new tab, but you'd also want the browser to _not_ automatically switch to it?  

I'm not sure this is a widely-desired UX change -- Never had many reports about it.  
Wouldn't the holding ctrl trick be enough for you?

**Comment by darkred3** [comment]
@Difegue I didn't know about this ctrl trick, thank you. Personally though, I still think LRR should mimick default browser behavior by opening new gallery tabs in background. It is what Firefox, Chrome and probably all other browsers do, so I assume new users would expect this behavior.

**Comment by Difegue** [comment]
Middle mouse click also works for this, btw.  

I'll convert this issue to a Discussion so others can see it and eventually chime on changing this behavior, but I currently think it should stay as-is: It'd certainly be [going against browser defaults.](https://stackoverflow.com/questions/7924232/how-to-open-new-tab-in-javascript-without-switching-to-the-new-tab)
